### PR TITLE
data: Olumarshal

### PR DESCRIPTION
**Pull request by Olumarshal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Olumarshal.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Marshal Olu",
+  "bio": "Node.js Backend Developer | AltSchool Alumni | **OpenSource** Contributor",
+  "tags": [
+    "Open Source",
+    "JavaScript",
+    "TypeScript",
+    "Cloud Computing",
+    "Devops",
+    "Databases",
```
Comment (SaraJaoude) [inline data/Olumarshal.json:10]: remove comma:
```suggestion
    "Databases"
```

File: data/Olumarshal.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "url": "https://twitter.com/Ng_Marshal"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/Olumarshal"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/olumarshal"
+    },
```
Comment (SaraJaoude) [inline data/Olumarshal.json:24]: remove comma:
```suggestion
    }
```

File: data/Olumarshal.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/Ng_Marshal",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
```
Comment (SaraJaoude) [inline data/Olumarshal.json:33]: remove comma:
```suggestion
    }
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for contributing! 🎉

**Pull request merged**